javascripts/pop-up.js

```diff
@@ -0,0 +1,141 @@
+const projectDetails = [
+  {
+    name: 'Keeping track of hundreds of components #1',
+    deskcription: [
```

Comment:
- [ ] **[OPTIONAL]** You might want to fix this small typo in order to maintain **consistency and professionalism** in your project. 🙌

Reply:
I could not find this typo, kindly would you please mention where is it? @taldr27

Reply:
- [ ] **[OPTIONAL]** Sure, it's inside the `pop-up.js` file, make sure to change it in all the objects inside your `projectDetails` array. 🎯

  ![image](https://github.com/M-Anwar-Hussaini/Portfolio/assets/107952724/5c3c39ff-53de-4708-a889-eb3553717774)

Reply:
Ok, Thank you very much. I'm going to solve it. 
You are a very passionate and kind code reviewer, I honestly appreciate your work.
@taldr27